scrapers/GloryHoleSwallow.yml

```diff
@@ -0,0 +1,25 @@
+name: "GloryHoleSwallow"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://gloryholeswallow.com/newtour/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Details: //span[@class="latest_update_description"]/text()
+      Date:
+        selector: //ul/comment()[contains(.,'update_date')]
+        replace:
+          - regex: .+(?:<.+>)(.+)(?:<.+>)
+            with: $1
+        parseDate: January 02, 2006
```

Comment:
Dates used are in the 3 letter form
`January` should be switched to `Jan`

Reply:
Done!